### PR TITLE
Config active storage

### PR DESCRIPTION
**Pull request by LeandroSoria2**
### What changes does this PR include?
**Description**
Add active storage gem and configurations
**Summary**
- Add active storage gem
- Add configuration files
- Create a bucket in s3 

[Trello](https://trello.com/c/ixRPfzet/27-configure-amazon-s3-for-images)

**Preview**
Screen Shot 2022-07-22 at 16 36 50

### CODE REVIEW

**Comment by AgostinaDev** [comment]
Looks like this gem is missing:`aws-sdk-s3`

File: config/storage.yml [hunk 1]
```diff
@@ -7,28 +7,9 @@ local:
...
-# mirror:
-#   service: Mirror
-#   primary: local
-#   mirrors: [ amazon, google, microsoft ]
+amazon:
+  service: S3
+  access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
+  secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
+  bucket: <%= ENV['S3_BUCKET_NAME'] %>
+  region: <%= ENV['AWS_BUCKET_REGION'] %>
```
Comment (AgostinaDev) [inline config/storage.yml]: missing line at end of file

File: config/environments/development.rb [hunk 2]
```diff
@@ -34,7 +34,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :amazon
```
Comment (AgostinaDev) [inline config/environments/development.rb]: We usually configure `:local/:test` for `non-production` environments and `:amazon` for the `production` environment

File: config/storage.yml (hunk 1 above)
Comment (AgostinaDev) [reply config/storage.yml]: You'll probably also need to configure local/test envs like this:

https://github.com/rootstrap/rails_api_base/blob/master/config/storage.yml

File: config/environments/development.rb (hunk 2 above)
Comment (AgostinaDev) [reply config/environments/development.rb]: Here's an example for a production environment:
https://github.com/rootstrap/rails_api_base/blob/master/config/environments/production.rb#L46

Here's an example for a development environment:
https://github.com/rootstrap/rails_api_base/blob/master/config/environments/development.rb#L37

Here's an example for a test environment configuration:
https://github.com/rootstrap/rails_api_base/blob/master/config/environments/test.rb#L37

File: config/storage.yml (hunk 1 above)
Comment (LeandroSoria2) [reply config/storage.yml]: Fix in next commit

**Review by AgostinaDev — APPROVED** [review]
LGTM

File: .rubocop.yml [hunk 3]
```diff
@@ -37,3 +37,7 @@ Style/HashSyntax:
 Lint/UnusedMethodArgument:
   Exclude:
     - spec/support/devise_token_auth_overrides.rb  
+
+Metrics/MethodLength:
+  Exclude:
```
Comment (JulianPasquale) [inline .rubocop.yml:43]: Can we exclude `db/migrate` for all the rules?

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,16 +0,0 @@
-<!DOCTYPE html>
```
Comment (JulianPasquale) [inline app/views/layouts/application.html.erb:1]: Not sure why this is being deleted, but doesn't sound like something you should do in this PR. If the PR title is "Config Active Storage" then we should only include the changes for that

File: .rubocop.yml (hunk 3 above)
Comment (LeandroSoria2) [reply .rubocop.yml:43]: Yes, I will change it

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (LeandroSoria2) [reply app/views/layouts/application.html.erb:1]: I will keep this recommendation in mind for future pr

File: .rubocop.yml (hunk 3 above)
Comment (JulianPasquale) [reply .rubocop.yml:43]: Since we exclude the entire directory, we should remove this from here

**Review by JulianPasquale — APPROVED** [review]

**Pull request merged**